src/main/java/org/broadinstitute/consent/http/authentication/DuosUserAuthorizer.java

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.consent.http.authentication;
+
+import io.dropwizard.auth.Authorizer;
+import jakarta.ws.rs.container.ContainerRequestContext;
+import org.broadinstitute.consent.http.db.UserRoleDAO;
+import org.broadinstitute.consent.http.models.DuosUser;
+
+public class DuosUserAuthorizer extends AuthorizerHelper implements Authorizer<DuosUser> {
```

Comment:
I think these subclasses can be avoided by making `AuthorizerHelper` a generic class. So where you're writing `new DuosUserAuthorizer()` you could write `new AuthorizerHelper<DuosUser>()`.

Reply:
I like that idea. That might let me simplify the other abstract class too.